deploy.sh

```diff
@@ -57,22 +63,14 @@ function retry_terraform {
     cmd=$2
     for i in $(seq 1 "$tries"); do
         fatal_errors=()
-        LOGFILE=$LOGFILE_$i
+        LOGFILE=${LOGFILE}_$i
         echo "Attempt: $i/$tries"
         echo "========================" >>$LOGFILE
         echo "Attempt: $i/$tries" >>$LOGFILE
         echo "$cmd" >>$LOGFILE
         echo "========================" >>$LOGFILE
-        $cmd >>$LOGFILE 2>&1 &
```

Comment:
Can we keep the run in background.. and keep the while loop for monitoring the progress? do you see any problem with it?
Initial idea of having it in background was to do other plumbing work while terraform is running. eg: reboot machine if not accessible.

Reply:
- we can keep running in the background
- while loop was giving issue on RHEL (will check how it can be made workable)